github-flow-mirror-repo/dist/index.js

```diff
@@ -65562,6 +65574,9 @@ on:
   push:
     branches:
       - main
+  schedule:
```

Comment:
Could this use a random number between 0 and 59 in the code generating this somehow - so that all Github actions don't all kick off at the same time for all mirror repos created?

Reply:
Sure we can discuss that and have it as a feature in the future work.